airbyte-webapp/src/packages/cloud/views/credits/CreditsPage/components/RemainingCredits.tsx

```diff
@@ -28,7 +28,7 @@ interface Props {
 const Block = styled.div`
   background: ${({ theme }) => theme.blue50};
   border-radius: 8px;
-  padding: 18px 25px 22px;
+  padding: 15px 20px;
   font-size: 13px;
   line-height: 20px;
   display: flex;
```

Comment:
Worth converting this away from styled-components, use our actual variables, etc. ... but in the interest of getting this merged this AM that will be a separate PR!